docs/mosaicking.rst

```diff
@@ -0,0 +1,310 @@
+    ...                                                 resolution=1.5 * u.arcsec)
+
+Projection and reference coordinate
+-----------------------------------
+
+Finally, you can customize the projection to use as well as the reference
+coordinate. To change the projection from the default (which is the
+gnomonic projection, or ``TAN``), you can use the ``projection=`` keyword
+argument, which should be set to a valid three-letter FITS-WCS projection code::
+
+  >>> wcs_out, shape_out = find_optimal_celestial_wcs(m17_hdus,
+  ...                                                 projection='CAR')
+
+To customize the reference coordinate (where the projection is centered) you
+can set the ``reference=`` keyword argument to an astropy
+:class:`~astropy.coordinates.SkyCoord` object::
+
+    >>> from astropy.coordinates import SkyCoord
+    >>> coord = SkyCoord.from_name('M17')
+    >>> wcs_out, shape_out = find_optimal_celestial_wcs(m17_hdus,
+    ...                                                 reference=coord)
+
+.. _coadding:
+
+Reprojecting and co-adding images into a mosaic
+===============================================
+
+Assuming that you have a set of images that you want to combine into a mosaic,
+as well as a target header or WCS and shape (which you either determined
+independently, or with :ref:`optimal-wcs`), you can make use of the
+:func:`~reproject.mosaicking.reproject_and_coadd` function to produce the
+mosaic::
+
+    >>> from reproject import reproject_interp
+    >>> from reproject.mosaicking import reproject_and_coadd
+    >>> array, footprint = reproject_and_coadd(m17_hdus,
+    ...                                        wcs_out, shape_out=shape_out,
+    ...                                        reproject_function=reproject_interp)
+
+The first argument to :func:`~reproject.mosaicking.reproject_and_coadd`
+should be a list where each element is either a filename, an HDU object (e.g.
+`~astropy.io.fits.PrimaryHDU` or `~astropy.io.fits.ImageHDU`), an
+`~astropy.io.fits.HDUList` object, or a tuple of ``(array, wcs)``.
+
+The second argument is the WCS information for the output image, which should
+be specified either as a :class:`~astropy.wcs.WCS` or a
+:class:`~astropy.io.fits.Header` instance. If this is specified as a
+:class:`~astropy.wcs.WCS` instance, the ``shape_out`` argument to
+:func:`~reproject.reproject_interp` should also be specified, and be
+given the shape of the output image using the Numpy ``(ny, nx)`` convention
+(this is because :class:`~astropy.wcs.WCS`, unlike
+:class:`~astropy.io.fits.Header`, does not contain information about image
+size).
+
+Finally, the ``reproject_function`` should be used to specify which function to
+use to reproject individual tiles - this should be either
+:func:`~reproject.reproject_interp` or :func:`~reproject.reproject_exact` - with
+the latter being slower but more accurate. Keyword arguments for these functions
```

Comment:
maybe emphasize `"...accurate and necessary for flux conservation."` ?

Reply:
Done